pkg/check/node_cbt.go

```diff
@@ -0,0 +1,108 @@
+package check
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/vmware/govmomi/vim25/mo"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/component-base/metrics"
+	"k8s.io/component-base/metrics/legacyregistry"
+	"k8s.io/klog/v2"
+)
+
+// CollectNodeCBT emits metric with CBT Config of each VM
+type CollectNodeCBT struct {
+	lastMetricEmission map[bool]int
```

Comment:
It is bit odd to see use of boolean as keys for a map. I understand that, this is for tracking number of nodes which has cbt enabled and number of nodes which has cbt disabled. Can we use strings here?

Reply:
I can switch it.  would "true"/"false" still be ok for the values stored?